core/src/main/java/org/elasticsearch/index/mapper/geo/GeoPointFieldMapper.java

```diff
@@ -123,11 +126,45 @@
 
         private int geoHashPrecision = Defaults.GEO_HASH_PRECISION;
 
+        private Boolean ignoreMalformed;
+
+        private Boolean coerce;
+
         public Builder(String name) {
             super(name, Defaults.FIELD_TYPE);
             this.builder = this;
         }
 
+        public Builder ignoreMalformed(boolean ignoreMalformed) {
+            this.ignoreMalformed = ignoreMalformed;
+            return builder;
+        }
+
+        protected Explicit<Boolean> ignoreMalformed(BuilderContext context) {
+            if (ignoreMalformed != null) {
+                return new Explicit<>(ignoreMalformed, true);
+            }
+            if (context.indexSettings() != null) {
```

Comment:
Hrm, I hate adding another use looking up an index setting within mappings. I know the other field mappers do this, but I was hoping to remove this setting sometime soon. @clintongormley do you think we need to support this here?